### ISSUE TITLE
Diggy Scenario Idea - Abandoned outposts(loot base)

### ISSUE THREAD

**Issue by naquadah** [Valansch/RedMew#245]
**Very rare chance of spawning an abandoned mining outpost(loot base)**

- Very rare chance of spawning or rare for big outposts and more common for small ones
- Small ones could be as simple as small half broken track with a cargo wagon on it. or just a random car with  some chests and burner drills around it.
- Or auto generated on map exploration in rare 
- Bit more fancy designed than the crash site ones ( no big piles of square chests of loot.)

**Comment by iltar** [comment]
@Valansch had a nice idea for this (picture taken from Discord)
![image](https://user-images.githubusercontent.com/1754678/48275379-a3654700-e445-11e8-842d-195df45db88c.png)

I'm currently working on an artefact hunting system that will spawn treasure chests. Hopefully we can also add something like outposts as well

**Comment by naquadah** [comment]
Yeah nice. I was supposed to help create some layouts. I got some free time this weekend so i can look into it then.
